docs/doc_v1.0/Build_documentation.md

```diff
@@ -121,20 +145,63 @@ go on in timesteps.
 Infected: the number of the population initially infected at t=0. Must
 be greater than 0.
 
-Seed: the seed for the random number generator. Will be modified by the
-command line argument if included. This ensure a reproducible output.
-
 Steady state: if there are still active infections when the contact data
 has been processed, the steady state flag enables the active cases to
 reach a resolution (recovered or dead) with no further random infections
 occurring.
 
-Contacts file: points to the CSV file being used in this run.
+(Optional) Seed: Can be specified on the command line or fixed.
 
 ![](image5.png)
 
 Figure 5. the run settings file
 
+
+## AgeData
+
+The age data file contains a list of id and age. This has been added 
+to allows the user to use age graduated contact files. Should the file 
+be longer than the population, additional input will be ignored. Should 
+it be shorter, the ages will be generated as per the population settings file. 
```

Comment:
I think it's not really about how long the age file input is... I think the distinction is more to do with whether or not the id exists in the age data file. If the id exists we will use the provided age, otherwise we would generate it as per the population settings file